packages/common/src/interfaces.ts

```diff
@@ -118,3 +123,22 @@ export interface HistoryAndWorkflowId {
   workflowId: string;
   history: temporal.api.history.v1.History | unknown | undefined;
 }
+
+/**
+ * Policy defining actions taken when a workflow exits while update or signal handlers are running.
+ * The workflow exit may be due to successful return, failure, cancellation, or continue-as-new.
```

Comment:
You mention the different ways that a workflow may exit. We should make it clear (probably below, on `WARN_AND_ABANDON`) that failure dont result in warn.

Reply:
Good call, done.